app/services/lifetime_usages/recalculate_and_check_service.rb

```diff
@@ -31,7 +31,7 @@ def progressive_billed_amount
       subscription.invoices
         .finalized
         .progressive_billing
-        .sum { |invoice| invoice.fees.progressive_billing.sum(:amount_cents) }
```

Comment:
should we remove the `progressive_billing` fee type if we're no longer using it?

& 

can we change this to just be `fees_amount_cents`?